### ISSUE TITLE
Bug in navwalker

### ISSUE THREAD

**Issue by afterimagedesigns** [wp-bootstrap/wp-bootstrap-navwalker#428]
<h2>Incorrect data in title attribute and it’s causing problems in some cases.</h2>

A client once filed me a bug regarding with the title attribute issue. There seems a problem between these lines

`$atts['title'] = ! empty( $item->title ) ? $item->title : '';`

Where it should say:

`$atts['title'] = ! empty( $item->attr_title ) ? $item->attr_title : '';`

I'm not sure if this is right. Can you please check this on your end which code is right?

**Comment by IanDelMar** [comment]
It seems as this is a result of [this code](https://github.com/wp-bootstrap/wp-bootstrap-navwalker/blob/master/class-wp-bootstrap-navwalker.php#L173-L177)
```php
if ( empty( $item->attr_title ) ) {
	$atts['title'] = ! empty( $item->title ) ? strip_tags( $item->title ) : '';
} else {
	$atts['title'] = $item->attr_title;
}
```
Here the WP Bootstrap nav walker deviates from the WordPress nav walker. While the latter does not default to anything if no title attribute is specified, the first defaults to the menu item title.

**Comment by IanDelMar** [comment]
@pattonwebz Fixed in #468

**Comment by IanDelMar** [comment]
As of #468 no default value is set to the title attribute.
